src/dev-utils/enableenginedebug.js

```diff
@@ -317,27 +320,57 @@ function enableLoggingTools() {
 		}
 	};
 
+	Delta.prototype.saveHistory = function( before, transformedBy, wasImportant, resultIndex, resultsTotal ) {
```

Comment:
Also `before, transformedBy, wasImportant, resultIndex, resultsTotal` should be replaced by an object.